packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -302,4 +314,11 @@ export class FluidContainer extends TypedEventEmitter<IFluidContainerEvents> imp
         this.container.off("saved", this.savedHandler);
         this.container.off("dirty", this.dirtyHandler);
     }
+
+    /**
+     * {@inheritDoc IFluidContainer._getRuntimeContainer}
+     */
+    public _getRuntimeContainer(): IContainer {
```

Comment:
nit: i think runtime is the wrong word here. as we have a container runtime which is very different from this container. this container encapsulates a runtime

Reply:
I was unsure about the name when I added it. Really, I'm trying to capture the layering semantics here - I.e. the inner 1st party API container on this 3rd party API wrapper. "innerContainer" sounded to general. Definitely interested in suggestions.

Reply:
I'm going to rename to "_getInternalFoo" for the time being. I'll consider other naming options between now and when this aspect gets PR'd into main.